core/lib/multivm/src/glue/types/vm/vm_block_result.rs

```diff
@@ -227,6 +230,7 @@ impl GlueFrom<crate::vm_1_3_2::vm_instance::VmBlockResult>
                 circuit_statistic: Default::default(),
             },
             refunds: Refunds::default(),
+            new_known_factory_deps: Default::default(),
```

Comment:
Please don't use `Default::default()` instead of `None`; this is overly verbose and unnecessarily opaque.